shell/platform/tizen/channels/window_channel.cc

```diff
@@ -0,0 +1,80 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "window_channel.h"
+
+#include <system_info.h>
```

Comment:
Unused include?

Reply:
Sorry, forgot to remove it after removing:

```
system_info_get_platform_int("http://tizen.org/feature/screen.width", &width);
system_info_get_platform_int("http://tizen.org/feature/screen.height", &height);
```